orbstation/modules/cargo/packs.dm

```diff
@@ -0,0 +1,59 @@
+//CARGO CRATES FOR ORBSTATION
+
+/datum/supply_pack/medical/cadaver
+	cost = CARGO_CRATE_VALUE * 5
+	access = ACCESS_MEDICAL
+	crate_type = /obj/structure/closet/crate/coffin //this is temporary
+	crate_name = "medical cadaver crate"
+
+//this SHOULD function identically for every cadaver crate
+/datum/supply_pack/medical/cadaver/generate()
+	. = ..()
+	var/mob/living/carbon/human/H = locate() in .
+	H.death()
+	for (var/part in H.internal_organs) //each cadaver comes with a complimentary set of organs, held in stasis
+		var/obj/item/organ/O = part
+		O.organ_flags |= ORGAN_FROZEN
```

Comment:
Does this get removed automatically when the organ is manipulated or are they frozen forever?
It looks like the other places which apply this have to manually remove it again.

Reply:
You may need to make a subtype of `coffin` which applies and removes this flag on all organs when it is opened/closed.
You were going to eventually anyway, this just means not waiting for a sprite.